source/mupdf.dpp

```diff
@@ -1,508 +1,2 @@
 #include <mupdf/fitz.h>
 #include <mupdf/pdf.h>
-
-// Macros
```

Comment:
How are macros handled in this new version?

Even if just syntactic sugar, they are definitely very helpful to achieve simplicity and similar syntax as in C/C++.

Reply:
`--function-macros` d++ switch probably does this job

https://github.com/AlexAltea/mupdf-d/blob/e916764dfcc9f12d68e9f9f54ed825ccd3512d56/source/mupdf_posix.d#L13265

I am still not tried to use this library, only tests passed on one piece of software (you know about it)
So, for now it is just a draft.